### PR TITLE
Task/#161 website developer documentation

### LINKED ISSUE #172: Related links do not appear in category landing page

### LINKED ISSUE #173: Grammatical error in v1.0.x (alpha) documentation
Issue in v1.0.x (alpha) documentation

Page: https://carbonldp.com/documentation/v1.0.x/quick-start-guide/

2. Run a platform instance

Run the following command to start your own platform instance for the first time. The command will fetch the image from a Docker repository **and will run the an instance**.

Should be:

The command will fetch the image from a Docker repository **and will run the instance**.

Also, in this statement:

"If you want to learn what each of *this* parameters are indicating you can read the following section:"

"this" should be "these"

### PR DESCRIPTION
**Pull request by jgf20**
- Fix #172 related links do not appear in category landing page
- Fix #173 grammatical errors in quick start guide v1.0.x
- Complete #161 add website project documentation:
    - Update README.md
    - Add project developer-documentation directory, website-strcutre.md.
    - Move ccs-guide.md from wiki developer-documentation directory
- #132 check css and js files

### CODE REVIEW

File: developer-documentation/website-structure.md [hunk 1]
```diff
@@ -0,0 +1,579 @@
...
+- `source/_data` files
+
+We use hexo to generate our site, to know more about how hexo works please refer to their [documentation](https://hexo.io/docs/)
+
+After the pages are processed by hexo a package hexo-widgets further processes the pages to build the public site adding widgets defined by the user
+
+Hexo Widgets
+- `themes/CarbonLDP/widgets/` files
+
+# Page structure
```
Comment (MiguelAraCo) [inline developer-documentation/website-structure.md]: Markdown pages should only have one level 1 header

**Comment by jgf20** [comment]
Since the document has many subsections, changing all the headers 1 to headers 2, made it a little difficult to read. I opted for dividing the document in two others: that hold the documentation and tools sections, the headers of this sections remain in the original document(website-structure.md) with reference to the file that holds the information in question. (documentation-structure.md and widgets-and-tools.md).  Let me know your thoughts, I think this makes it easer to digest and it ensure we do not have a mile long file.

**Comment by alvaro9210** [comment]
The README for the developer-documentation folder was added. I think we can proceed to merge the PR.

**Comment by MiguelAraCo** [comment]
Please update this PR to merge it

**Pull request merged**